src/test/ui/binary-op-on-double-ref.stderr

```diff
@@ -4,8 +4,7 @@ error[E0369]: binary operation `%` cannot be applied to type `&&{integer}`
 LL |         x % 2 == 0
    |         ^^^^^
    |
-   = note: this is a reference to a type that `%` can be applied to; you need to dereference this variable once for this operation to work
-   = note: an implementation of `std::ops::Rem` might be missing for `&&{integer}`
+   = help: `%` can be used on '&{integer}', you can dereference `x`: `*x`
```

Comment:
This message is slightly wrong. I'm ok with it if you do not want to go down the rabbit hole of getting `{integer}` here instead, but we would have to follow up to fix it at some point.